apps/google-analytics-4/frontend/src/components/config-screen/GoogleAnalyticsConfigPage.tsx

```diff
@@ -169,8 +174,9 @@ export default function GoogleAnalyticsConfigPage() {
           isInEditMode={isInEditMode}
           onInEditModeChange={handleInEditModeChange}
           onIsValidServiceAccount={handleIsValidServiceAccount}
+          onHasServiceCheckErrorsChange={handleHasServiceCheckErrorsChange}
         />
-        {isAppInstalled && !isInEditMode && (
+        {isAppInstalled && (!hasServiceCheckErrors || parameters.propertyId) && (
```

Comment:
I think we've discussed this before and maybe it's a me problem, but  I personally think this is a pretty big cognitive lift to understand the conditional logic of this line. I would vote to extract it to a smaller helpful function where it could be commented (and edited) more easily.